index.html

```diff
@@ -288,8 +316,24 @@
 
   </g>
       
-      
+<!-- buttons -->
+  <g class="piggyButton">
+    <image x="50" y="430" width="100" height="80" 
+     xlink:href="http://static3.wikia.nocookie.net/__cb20110208032728/disney/images/6/68/Piglet_KH.png"/>
```

Comment:
images should be local to your project. If you depend on external assets, it may break without warning in the future